From 407f75a09434d50b8b96616758afa74feacc194d Mon Sep 17 00:00:00 2001
From: Alex Naidis <alex.naidis@linux.com>
Date: Fri, 28 Apr 2017 18:49:15 +0200
Subject: [PATCH 161/211] SnapdragonCamera: Add missing toast on HSR/HFR
 override

Add in a missing toast to warn a user when the wanted
HSR/HFR mode gets overriden due to the incompatibility
with image stabilization.

Change-Id: Ib81c50e8fad0bca075e6b33399321da42d77eb03
Signed-off-by: Alex Naidis <alex.naidis@linux.com>
---
 src/com/android/camera/VideoMenu.java | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/com/android/camera/VideoMenu.java b/src/com/android/camera/VideoMenu.java
index 418dbbe00..1caa1380d 100644
--- a/src/com/android/camera/VideoMenu.java
+++ b/src/com/android/camera/VideoMenu.java
@@ -808,6 +808,8 @@ public class VideoMenu extends MenuController
                 || !videoHDR.equals("off")
                 || timeLapseInterval != 0) {
             mListMenu.setPreferenceEnabled(CameraSettings.KEY_VIDEO_HIGH_FRAME_RATE, false);
+            RotateTextToast.makeText(mActivity, R.string.error_app_unsupported_hfr_selection,
+                    Toast.LENGTH_LONG).show();
         }
 
     }
-- 
2.17.1

